code/game/machinery/camera/camera.dm

```diff
@@ -358,6 +357,16 @@
 	busy = 0
 	return 0
 
+/obj/machinery/camera/proc/Togglelight(on=0)
+	for(var/mob/living/silicon/ai/A in world)
```

Comment:
Use the player list, "in world" should never be used for a single type of something, especially if it is already in another list!